index.php

```diff
@@ -350,15 +350,15 @@
         <img class="app-display__image" src="images/screenshots/shortcut-overlay.png" width="885" height="656" alt="elementary OS Keyboard Shortcuts"/>
         <div class="app-display__description">
           <h1>User Friendly. And Keyboard Friendly.</h1>
-          <p>elementary OS is designed to be easy to understand and pick up as a new user. But that doesn’t mean it’s dumbed down; powerful, customizable keyboard shortcuts ensure you’ll stay productive whether it’s your first day or your thousandth.</p>
+          <p>elementary OS is designed to be easy to understand and pick up as a new user. But that doesn’t mean it’s dumbed down; powerful, customizable keyboard shortcuts ensure you stay productive on your first or thousandth day.</p>
```

Comment:
```suggestion
          <p>elementary OS is designed to be easy to understand and pick up as a new user. The one-armed man can fight, and we offer a honed axe. Powerful, customizable keyboard shortcuts within reach to stay productive on your first or thousandth day.</p>
```
Another nod to the sagas.